agent/ae/ae_test.go

```diff
@@ -400,7 +400,7 @@ func testSyncer(t *testing.T) *StateSyncer {
 		InferLevels: true,
 	})
 
-	l := NewStateSyncer(nil, time.Second, nil, logger)
+	l := NewStateSyncer(nil, time.Second, nil, logger, consulLogger)
```

Comment:
For consistency did you want this to be `logger2` instead of `consulLogger` like you refactored in an earlier file?

Reply:
👍